docker-compose.yml

```diff
@@ -68,11 +69,24 @@ services:
     networks:
       - wca-main
 
+  wca_redis:
+    image: redis:7.0
+    restart: always
+    ports:
+      - "6379:6379"
+    command: redis-server --save 20 1 --loglevel warning
```

Comment:
This saves a snapshot of redis every 20 seconds if at least one operation was performed. Not really necessary for development though (it would allow for picking up with cache still full, but probably not needed)